test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/TxnVerbs.java

```diff
@@ -411,6 +417,17 @@ public static HapiContractCreate contractCreate(
         }
     }
 
+    public static HapiContractCreate contractCreate(
+            final String contractName, final String constructorParam) {
+        if (constructorParam != null) {
+            final var constructorABI = getABIFor(CONSTRUCTOR, EMPTY, contractName);
+            return new HapiContractCreate(contractName, constructorABI, constructorParam)
+                    .bytecode(contractName);
+        } else {
+            return new HapiContractCreate(contractName).bytecode(contractName);
+        }
+    }
```

Comment:
Since `Utils.asHexedAddress` returns `String`, this overloaded method becomes obsolete. It can be removed.